cocos/ui/UIEditBox/Mac/CCUIEditBoxMac.mm

```diff
@@ -135,6 +135,8 @@ - (NSWindow *)window
 - (void)openKeyboard
 {
     [self.window.contentView addSubview:self.textInput];
+    //FIXME: it will cause a few OpenGL error once, should be fix when glfw version upgraded.
```

Comment:
@dumganhar 
In the future, when glfw fixed the EditBox input issue on mac os, we should remove this hack code.